apisix/http/router/radixtree_sni.lua

```diff
@@ -95,6 +134,9 @@ local function create_router(ssl_items)
         end
     end
 
+    parsed_certs = core.table.new(0, idx)
+    parsed_pkeys = core.table.new(0, idx)
```

Comment:
Maybe we can store the parsed data in `ssl.value` directly, so the data can be reused even after router recreated.

Reply:
The current way is wrong. We cannot store them directly in the hash table. This method will cause memory overflow.

We need to cache the parsed data in lrucache, and we need to control the number of cached certificates.